src/Fixer/Basic/CurlyBracesPositionFixer.php

```diff
@@ -36,6 +36,11 @@ final class CurlyBracesPositionFixer extends AbstractFixer implements Configurab
 {
     use Indentation;
 
+    /**
+     * @internal
+     */
+    public const NEXT_LINE_IF_RETURN_TYPEHINT = 'next_line_if_return_typehint';
```

Comment:
```suggestion
    public const NEXT_LINE_IF_RETURN_TYPE_DECLARATION = 'next_line_if_return_type_declaration';
```
Some time ago we decided to get rid of "typehint" convention (see #7338 for example, but there are also deprecated fixers like `function_typehint_space` and `compact_nullable_typehint`), so I believe we should change the name of this option accordingly.

Sorry for long delay, I had tough time recently but slowly coming back to Fixer.